weblate/machinery/tests.py

```diff
@@ -1489,6 +1489,7 @@ class OpenAITranslationTest(BaseMachineTranslationTest):
         "model": "auto",
         "persona": "",
         "style": "",
+        "base_url": "https://api.openai.com/v1",
```

Comment:
Should not be needed:

```suggestion
```

On the other side, can you please add a separate test case to verify that `base_url` and custom model are actually being used?